### PR TITLE
Logo Test Pull Request

### PR DESCRIPTION
**Pull request by llucas314**
PR for  issue #229

### CODE REVIEW

File: client/src/components/Logo/Logo.test.js [hunk 1]
```diff
@@ -1,16 +1,43 @@
...
+    const wrapper = setUp({ clickable: true, to: "About" });
+    const logo = findByTestAttribute(wrapper, "Link");
+    expect(logo.prop("to")).toEqual("About");
+  });
+  it("Should render with all props", () => {
+    const wrapper = setUp({ clickable: true, to: "About", small: true });
+    const logo = findByTestAttribute(wrapper, "Link");
+    const small = findByTestAttribute(wrapper, ".Logo--small");
+    expect(logo.prop("to")).toEqual("About");
+    expect(small.length).toBe(1);
```
Comment (clarknoah) [inline client/src/components/Logo/Logo.test.js]: This all looks good, one question though is why the findByTestAttribute function was created? It seems a bit redundant. for example:
`const small = findByTestAttribute(wrapper, ".Logo--small");` is the same as
`const small = wrapper.find(".Logo--small");`

I would personally remove this function as I don't think it's needed. In my OPINION!

File: client/src/components/Logo/Logo.test.js [hunk 2]
```diff
@@ -1,16 +1,43 @@
-import React from 'react'
-import { shallow, configure } from 'enzyme'
-import Logo from './Logo.js'
-import Adapter from 'enzyme-adapter-react-16';
-configure({adapter: new Adapter()});
+import React from "react";
+import { shallow } from "enzyme";
+import Logo from "./Logo.js";
+import { findByTestAttribute } from "../../../../utils/utils";
```
Comment (clarknoah) [inline client/src/components/Logo/Logo.test.js]: I would remove this

File: client/src/components/Logo/Logo.test.js [hunk 3]
```diff
@@ -1,16 +1,43 @@
...
-    // We create an assertion within the test that checks if our component renders our name prop
-  // expect(component.contains('Your name')).toBe(true)
-  })
-})
+describe("Logo component", () => {
+  const setUp = props => {
+    const component = shallow(<Logo {...props} />);
+    return component;
+  };
+  it("Should render as expected", () => {
```
Comment (clarknoah) [inline client/src/components/Logo/Logo.test.js]: I really like this idea!

Comment (clarknoah) [inline client/src/components/Logo/Logo.test.js]: I really like this idea!

File: client/src/components/Logo/Logo.test.js [hunk 4]
```diff
@@ -1,16 +1,43 @@
...
+  it("Should render with clickable prop", () => {
+    const wrapper = setUp({ clickable: true });
+    const logo = findByTestAttribute(wrapper, "Link");
+    expect(logo.length).toBe(1);
+  });
+  it("Should render with clickable prop", () => {
+    const wrapper = setUp({ clickable: true });
+    const logo = findByTestAttribute(wrapper, "Link");
+    expect(logo.length).toBe(1);
+  });
```
Comment (clarknoah) [inline client/src/components/Logo/Logo.test.js]: What exactly is this testing for? Wouldn't you want something like this?

`expect(wrapper.props().clickable).toBe(true)`?

**Comment by llucas314** [comment]
Revisions corrected

**Pull request merged**